### PR TITLE
Clean up workflow definitions

### PR DESCRIPTION
**Pull request by OR13**
Move the definitions to the extensions of the standard section so they can be linked directly too.

Adding better examples.

### CODE REVIEW

**Review by BenjaminMoe — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -528,23 +528,57 @@ <h2>Terminology</h2>
...
-        <dd>
+      </dl>
+
+      <section>
+        <h3>Extensions to Standard</h3>
+
+        <section>
+          <h4>Verifiable Credential</h4>
+
+          <p>The object must conform to the basic requirements section of the <a href="https://www.w3.org/TR/vc-data-model/#basic-concepts">W3C VC Data Model</a>.</p>
```
Comment (TallTed) [inline docs/index.html:539]:
```suggestion
          <p>The object MUST conform to the basic requirements section of the <a href="https://www.w3.org/TR/vc-data-model/#basic-concepts">W3C VC Data Model</a>.</p>
```

File: docs/index.html [hunk 2]
```diff
@@ -528,23 +528,57 @@ <h2>Terminology</h2>
...
+
+          <p>The <code>@context</code> MUST contain <code>"https://w3id.org/traceability/v1"</code>.</p>
+
+          <p>The object MUST have an <code>id</code> property, and its value MUST be a valid IRI (URI, URN).</p>
+        </section>
+
+        <section>
+          <h4>Verifiable Presentation</h4>
+
+          <p>The object must conform to the basic requirements section of the <a href="https://www.w3.org/TR/vc-data-model/#basic-concepts">W3C VC Data Model</a>.</p>
```
Comment (TallTed) [inline docs/index.html:549]:
```suggestion
          <p>The object MUST conform to the basic requirements section of the <a href="https://www.w3.org/TR/vc-data-model/#basic-concepts">W3C VC Data Model</a>.</p>
```

File: docs/index.html [hunk 3]
```diff
@@ -528,23 +528,57 @@ <h2>Terminology</h2>
...
             In the context of this vocabulary, we consider the specifics of
             workflows out of scope. We acknowledge that different use cases may
             have complicated workflows, which may yield many individual
             presentations of credentials.
           </p>
+          
+          <p class="note">
+            There are several systems for workflow definitions, see 
+            <a href="https://www.omg.org/spec/BPMN/2.0/About-BPMN/" target="_blank">BPMN</a>,
+            <a href="https://en.wikipedia.org/wiki/YAWL" target="_blank">YAWL</a>,
```
Comment (TallTed) [inline docs/index.html:573]:
```suggestion
            There are several systems for workflow definitions. See 
            <a href="https://www.omg.org/spec/BPMN/2.0/About-BPMN/" target="_blank">BPMN</a>,
            <a href="https://en.wikipedia.org/wiki/YAWL" target="_blank">YAWL</a>, and/or
```

File: docs/index.html [hunk 4]
```diff
@@ -528,23 +528,57 @@ <h2>Terminology</h2>
...
             workflows out of scope. We acknowledge that different use cases may
             have complicated workflows, which may yield many individual
             presentations of credentials.
           </p>
+          
+          <p class="note">
+            There are several systems for workflow definitions. See 
+            <a href="https://www.omg.org/spec/BPMN/2.0/About-BPMN/" target="_blank">BPMN</a>,
+            <a href="https://en.wikipedia.org/wiki/YAWL" target="_blank">YAWL</a>, and/or
+            <a href="https://docs.github.com/en/actions/using-workflows/about-workflows" target="_blank">GitHub Workflows</a>.
```
Comment (mkhraisha) [inline docs/index.html:574]: does Github Workflows fit here? 
This section describes Supply chain workflows

**Review by mkhraisha — APPROVED** [review]

File: docs/index.html (hunk 4 above)
Comment (OR13) [reply docs/index.html:574]: it's just an example... so that folks who are less familiar with workflows can read more about what they are.

**Pull request merged**